vcpkg.json

```diff
@@ -21,7 +22,17 @@
     "openssl",
     "rapidjson",
     "spdlog",
-    "zlib"
+    "zlib",
+    "azure-storage-blobs-cpp",
+    "azure-identity-cpp"
```

Comment:
These aren't needed right?

Reply:
azure-identity is needed for the example code, but the blobs aren't. How should we handle that?

Reply:
I'm not sure. Vcpkg must have some way to override this on the command line or something. It nothing else we can use manifest features like we do in vw. But this shouldn't make it into the default set.

Reply:
I'll look into ways to remove conditionally remove it in the manifest